.github/workflows/packages.yaml

```diff
@@ -10,8 +10,40 @@ on:
   workflow_dispatch:
 
 jobs:
-  build-distro-packages:
-    name: build packages
+  build-distro-packages-arm64:
+    runs-on: ubuntu-latest
+    name: build arm64 packages
+    strategy:
+      fail-fast: true
+      matrix:
+        format: [ rpm, deb ]
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Old version - update to v3 (same for the other one in the workflow).

Reply:
solved